hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/RSProcedureDispatcher.java

```diff
@@ -471,9 +474,10 @@ public ServerOperation(RemoteProcedure remoteProcedure, long procId, Class<?> rs
       this.rsProcData = rsProcData;
     }
 
-    public RemoteProcedureRequest buildRequest() {
+    public RemoteProcedureRequest buildRequest(long initiatingMasterActiveTime) {
```

Comment:
I prefer we add initiatingMasterActiveTime as a field of the operation, instead of a method parameter.We can add a initiatingMasterActiveTime in ServerOperation and RegionOperation class.

Reply:
@Umeshkumar9414 FYI

Reply:
At that time I had some thoughts in my mind - 
1. Including this in operation will introduce too many changes.
2. Should I add this to the Procedure class or in the Operation? But now I am clear in this perspective. I am adding this into operation as this is something related to dispatching not related to the procedure itself.

Reply:
There were other reasons also like there is a difference in how we create requests for OpenRegion and CloseRegions.  For OpenRegionRequest we create only one request for all the OpenRegionOperations but for close we create CloseRegoinRequest for each operation.
It would be all good in close but for OpenRegionRequest [here](https://github.com/apache/hbase/pull/6129/files#diff-f8d7389a8459b81b822e8ebbc129189771c1e203ac9268e3ee003d8131de9343R447), we need to get the activeCode from one of the entry.

Reply:
It would be like this 
```
builder.setInitiatingMasterActiveTime(operations.stream().map(
      RemoteOperation::getInitiatingMasterActiveTime).findAny().get());
```